src/fontra/client/web-components/glyphs-search.js

```diff
@@ -68,8 +69,8 @@ export class GlyphsSearch extends UnlitElement {
       { key: "glyphName", title: "glyph name", width: "10em", isIdentifierKey: true },
       {
         key: "unicode",
-        width: "5em",
-        get: (item) => makeUPlusStringFromCodePoint(item.unicodes[0]),
+        width: "fit-content",
+        get: (item) => makeUPlusStringFromCodePoints(item.unicodes),
```

Comment:
I would just inline the whole thing:
```suggestion
        get: (item) => item.unicodes?.map(makeUPlusStringFromCodePoints).join(","),
```
Your `makeUPlusStringFromCodePoints()` is too flexible in what it accepts and therefore more complicated than it should be.

Reply:
Thanks for the great feedback. Please see the new commits